### ISSUE TITLE
Would you consider switching to native technologies?

### ISSUE THREAD

**Issue by DerKnerd** [Superjo149/auryo#146]
Your app looks really good from the optics but is still far away from the native Look and Feel of Windows, macOS or any Linux DE. Would you consider writing a version that is native? It should be possible utilising [Proton native](https://proton-native.js.org/#/). It would be really awesome, cause I am looking for a native SoundCloud app for Linux since about six months that is actually native. And currently I have no time to build one myself...

**Comment by Superjo149** [comment]
No, i'm not planning to do this. Going for a native look wasn't my intention for this app. I looked at the different options before I started, but I wanted to add some of my own style to it. 

BTW, be on the lookout for a big update between this & a few days.

**Comment by DerKnerd** [comment]
@Superjo149 I understand that, I did that too a long time ago, when I was still writing desktop apps. But would you at least consider to drop electron? It drains the battery a lot. And btw. I am application designer and your design is pretty good so thumbs up for the good work 👍

**Comment by Superjo149** [comment]
Thanks, I appreciate the feedback. I understand native is better for performance but I went for electron for the ease of development & the javascript stack. I do not have the intention of rewriting to native to get better performance. Someone with experience in native app development can do that. I already spent way too many hours on this. 😛 

I know there are some performance issues, part of those are just related to my coding and the fact that I didn't know what I was doing back when I started. What I can do and I did (in the next update... which may be released today or tomorrow) is try and fix those. I personally have noticed around 3x less CPU usage on my machine, so I hope you'll be able to see a difference.
